.changelog/4476.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Moved PR100 and XD100 validations to run as a pre-commit hook. These validations ensure the ParsingRule and XSIAM Dashboard filenames are valid.
```

Comment:
mention the error codes for both, see other similar release notes and match their format.